From 31988f1cb9c292f299606455443f7a08e86534d9 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Fri, 12 May 2017 14:26:38 +0800
Subject: [PATCH] arm: rockchip_defconfig: enable CONFIG_KEYBOARD_GPIO

This adds enable CONFIG_KEYBOARD_GPIO to support gpio-keys driver.

Change-Id: Ib2e127a3d017ad69b1bf6c0b0a795d0bce44af0e
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 04fdf7170665..687c0b08ef9d 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -309,6 +309,7 @@ CONFIG_RTL8188EU=y
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_KEYRESET=y
 # CONFIG_KEYBOARD_ATKBD is not set
+CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_ROCKCHIP=y
 # CONFIG_INPUT_MOUSE is not set
 CONFIG_INPUT_JOYSTICK=y
-- 
2.35.3

